lib/mix/tasks/ash_phoenix.gen.live.ex

```diff
@@ -20,8 +20,8 @@ if Code.ensure_loaded?(Igniter) do
     ## Options
 
     * `--domain`   - Existing domain
-    * `--resource` - Existing resource
-    * `--resourceplural` - Plural resource name
+    * `--resource` - Existing resource module name
+    * `--resourceplural` - Pluralized version resource name in english
```

Comment:
It doesn't necessarily need to be in English does it? Its just whatever plural you'd like to use.

Reply:
Okay true. I’ll update